src/config.py

```diff
@@ -301,6 +307,67 @@ def load(self, load_include_dirs=True):
         """
 
         try:
+            PULL_SERVER_SIDE_CONFIG_PARAM = 'True'
+            # Read configuration files from default directories
+            config_files = [self.config_filename]
+            if load_include_dirs:
+                config_files.extend(self._list_configs(self.config_d))
+            self._set_config_file_perms(config_files)
+
+            # Read in json config file
+            with open(config_files[0]) as data_file:
```

Comment:
When load_json is called, it has already been established that a json config file exists - see load() method.
In this case we only have one json config file, so even though config_files extends the files in the directory, we only need the first element of the list which is the json config file by default (see config_filename).